docs/examples/basic-requirements.md

```diff
@@ -33,27 +34,27 @@ export PATH=$PATH:~/bin
 mv kubectl kops  ~/bin
 ```
 
-Finally, some of our exercises use the "jq" utility that is available on modern linux distributions. Please ensure to install it too. Some examples of how to do it:
+Finally, some of our exercises use the `jq` utility which is available on modern linux distributions. Please ensure to [install](https://stedolan.github.io/jq/download/) it as well.

```

Comment:
@MoShitrit could you also delete all the text below this line?

Reply:
Done 😄 
Thanks for reviewing @hakman !

Reply:
Np. Thanks for finding these.
@olemarkus does it look better now?